.gitmodules

```diff
@@ -16,3 +16,7 @@
 [submodule "speech_recognition/whisper/whisper"]
 	path = speech_recognition/whisper/whisper
 	url = https://github.com/AmpereComputingAI/whisper.git
+[submodule "utils/nlp/transformers"]
+	path = utils/nlp/transformers
```

Comment:
this branch of transformers is dedicated to llama only so maybe better place for it would be nlp/text_gen/alpaca/

Reply:
We're pretty inconsistent in that matter. We have nnUNet, dlrm and DeepCTR submodules in their respective `utils/<task>` directories, but segment_anything, nanoGPT and whisper submodules are in the `<task>/<model>` directories.

Reply:
that depends on whether datasets make use of submodule facilities or just the model runner